README.rst

```diff
@@ -1,47 +1,66 @@
-FreeType high-level python API
-==============================
+FreeType (high-level Python API)
+================================
 
-Freetype python provides bindings for the FreeType library. Only the high-level API is bound.
+Freetype Python provides bindings for the FreeType library. Only the
+high-level API is bound.
 
 Documentation available at: http://freetype-py.readthedocs.org/en/latest/
 
 Installation
 ============
 
-To be able to use freetype python, you need the freetype library version 2
-installed on your system.
+**From PyPI, recommended**: `pip install freetype-py`. This will install the
+library with a bundled FreeType binary, so you're ready to go on Windows,
+macOS and Linux (all with 32 and 64 bit support).
+
+Installation with compiling FreeType from source
+------------------------------------------------
+
+If you don't want to or can't use the pre-built binaries, build FreeType
+yourself: `export FREETYPEPY_BUNDLE_FT=yesplease && python setup.py install`.
```

Comment:
`pip install .` is the currently recommended method for installing from a source folder. The `setup.py install` method uses setuptools to create and install a (now deprecated) `egg` package, whereas `pip install .` will make a temporary `wheel` and install that with pip.
`pip install .` is also more future proof because build-system agnostic. In the future, one may use a different build tool than setuptools, and the next pip version will allow to specify custom build requirements.